doc/sw-test/sw_test.trlc

```diff
@@ -10,7 +10,12 @@ section "SW-Test" {
     }
 
     SwTestCase tc1 {
-        description = "Test"
+        description = "This test case ..."

```

Comment:
Is this a template only to be filled in later ?

Reply:
Yes, I needed them to see whether the toolchain works.